WireAPI/Tests/WireAPITests/UpdateEventsAPITests.swift

```diff
@@ -0,0 +1,97 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SnapshotTesting
+@testable import WireAPI
+import XCTest
+
+final class UpdateEventsAPITests: XCTestCase {
+
+    // MARK: - Request generation
+
+    func testGetLastUpdateEvent() async throws {
+        let snapshotter = APISnapshotHelper { httpClient, apiVersion in
+            UpdateEventsAPIBuilder(httpClient: httpClient)
+                .makeAPI(for: apiVersion)
+        }
+
+        try await snapshotter.verifyRequestForAllAPIVersions { sut in
+            _ = try await sut.getLastUpdateEvent(selfClientID: Scaffolding.selfClientID)
+        }
+    }
+
+    // MARK: - Response handling
+
+    // MARK: - V0
```

Comment:
As V5 has it's own implementation, the tests should be duplicated to ensure the implementation works.

Reply:
done a9d609098334302350b679e1656c084186fc25cf